### ISSUE TITLE
So many image links are invalid

### ISSUE THREAD

**Issue by gongwk** [Tencent/tencent-ml-images#12]
I run the mutithreading shell in windows with virtual linux env， but the terminal output shows that so many image urls are invalid. 1218 images in total, but i just download 228. 
![image](https://user-images.githubusercontent.com/14072360/47278657-2f314380-d5fd-11e8-956e-d3fbbe9df8cf.png)
In addtion，in 228 downloaded images， some images are still invalid.
![image](https://user-images.githubusercontent.com/14072360/47278705-9949e880-d5fd-11e8-878b-acef2f9a11eb.png)
how can I solve this problem and get enough images to do some research.

**Comment by wubaoyuan** [comment]
@gongwk The reason is that the URLs from ImageNet are too old, thus many are invalid. As suggested in our README, you can download the whole database of ImageNet (http://image-net.org/download), then collect the URLs used in our database, according to 'train_urls_and_index_from_imagenet.txt'. 
For other URLs from Open Images, you can download them directly.

**Comment by gongwk** [comment]
@wubaoyuan I‘ll try， thanks.

**Comment by wubaoyuan** [comment]
@gongwk Please see the latest README, there is a more clear demonstration about the invalid URLs

**Comment by Zeitzmz** [comment]
> @wubaoyuan I‘ll try， thanks.

Do you download all image successfully? Dose all invalid image url can be find in Imagenet?

**Comment by wubaoyuan** [comment]
@gongwk yes, please see "Download the original images of the whole database of ImageNet from (http://image-net.org/download), and the corresponding URL file is List of all image URLs of Fall 2011 Release (see http://image-net.org/download-imageurls)"

**Comment by wubaoyuan** [comment]
@Zeitzmz YES, please see "Download the original images of the whole database of ImageNet from (http://image-net.org/download), and the corresponding URL file is List of all image URLs of Fall 2011 Release (see http://image-net.org/download-imageurls)"
